### ISSUE TITLE
Enable computed gotos in the VM on compilers that support it

### ISSUE THREAD

**Issue by Gundolf68** [ThakeeNathees/pocketlang#111]
This gives a noticeable performance boost.

**Comment by ThakeeNathees** [comment]
Yeah, It's a great idea, That'll boost the performance a considerable amount. Thanks for the suggestion, I'll add this to my TODOs.

(Note for anyone willing to open a PR: First of all thanks but, because this is an architectural change, open an issue discussing the implementation first)
